src/dotnet/APIView/APIViewWeb/Languages/CSharpLanguageService.cs

```diff
@@ -72,6 +72,7 @@ public override Task<CodeFile> GetCodeFileAsync(string originalName, Stream stre
                             using var nuspecStream = entry.Open();
                             var document = XDocument.Load(nuspecStream);
                             var dependencyElements = document.Descendants().Where(e => e.Name.LocalName == "dependency");
```

Comment:
Is it possible to get this enumeration as empty list?

Reply:
It shouldn't be possible - everything will depend on at least core, and core has dependencies.

Reply:
I added a check though